### DISCUSSION TITLE
MultiSelect Field: No way to stop default selections from over writing replicated selections

### DISCUSSION THREAD

**Discussion by electrictomcat** [laravel/nova-issues#4489] [Q&A]
- Laravel Version: 9.18.0
- Nova Version: 4.10.1
- PHP Version: 8.1.7
- Database Driver & Version: mysq: 8.0.29 for macos12.2 on arm64

### Description:
I am using resolveUsing() to populate the values of a MultiSelect field when updating a resource, and I am also using different logic in default() when creating a new resource.

When you replicate a resource the default values of the MultiSelect field will over-write the replicated values.
This seems to occur because both resolveUsing and default will run for replication.

Can there be a toggle on default() to say notOnReplicate()?

### Detailed steps to reproduce the issue on a fresh Nova installation:
Create a MultiSelect field using both resolveUsing and default.
Duplicate a resource using that field, and the default values will override the replicated values.

**Comment by crynobone** [comment]
You should override the `replicate()` method and set the default model attribute: https://nova.laravel.com/docs/4.0/resources/#resource-replication

**Reply by electrictomcat** [reply]
Even if I hardcode values in the replicate() command they are still overwritten by default when replicating.

**Reply by crynobone** [reply]
> Even if I hardcode values in the replicate() command they are still overwritten by default when updating.

When `updating`? `default()` is only used on Create, Attach or Action page.

**Reply by electrictomcat** [reply]
When replicating Nova appears to handle it as a creation so defaults apply.
Sorry the previous comment was misleading, I am only concerned with replication behaviour.
